content/doc/administration/requirements/windows.adoc

```diff
@@ -7,18 +7,17 @@ This page documents the Windows support policy for the Jenkins server and agents
 
 == Scope
 
-Jenkins plugins, e.g. plugin:windows-slaves[WMI Windows Agents],
+Jenkins plugins, e.g., plugin:windows-slaves[WMI Windows Agents],
 may set additional requirements to Windows versions on controllers and/or agents.
-This page does not document such requirements,
-please refer to plugin documentation.
+This page does not document such requirements. Please refer to plugin documentation.
```

Comment:
Optional.  There is a general preference for sentence per line to simplify review comments.  This change is exactly the correct change, just a note that you might consider sentence per line in future pull requests.
```suggestion
This page does not document such requirements. Please refer to plugin documentation.
```

Reply:
OK, I will keep in my mind :)